packages/nimble-components/src/combobox/specs/IxD.md

```diff
@@ -0,0 +1,212 @@
+# Nimble Combobox (IxD)
+
+## Overview
+The combobox component is used to present a pre-defined list of options that end-users can choose from or type to create a custom value. Comboboxes combine the flexibility of text input with the efficiency of selection for more user-friendly interactions.
+
+> THE VISUAL DESIGN IN THIS DOCUMENT MAY NOT BE ACCURATE
+
+### Background
+- [Select and Combobox Refresh HLD (#1303)](https://github.com/ni/nimble/issues/1303)
+- [ViD Combobox Styles (Figma)](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1529-41777&mode=design)
+
+## Usage
+When to use:
+- To present a pre-defined list of options
+- To allow end-users to add custom values and/or options
+- For entering settings or preference values that don't (yet) exist
+- For single selection
+
+When not to use:
+- When the options are well-known and straightforward
+- When there are fewer than three options
+- For complex and/or hierarchical data
+- For multi selection
+
+### Anatomy
+![ ](spec-images/combobox-1.png)
+
+| Element | Description |
+| ---------- | ---------- |
+| Label | Text that tells the end-user what to expect in the component | 
+| Helper text | Text that provides extra context or instruction about what the end-user should select or enter | 
+| Field input | Input that displays the current selected or entered option | 
+| Menu button | Click area that opens the menu | 
+| Menu | A list of options to choose from displayed as an *open* state | 
+| Menu item | A selection the end-user can make, shown with other options in the menu | 
+
+#### Label
+Comboboxes should always have a label, unless approved by an interaction designer or user researcher (usually in toolbar use cases). *See [Select](/packages/nimble-components/src/select/specs/IxD.md) for more information.*
+
+#### Helper Text
+Only include helper text to provide information that aids the end-user in choosing the right selection. *See [Select](/packages/nimble-components/src/select/specs/IxD.md) for more information.*
+
+#### Field Input
+Comboboxes can be empty by default or have a default selection from the menu. Empty comboboxes should include placeholder text that propts the end-user to enter a custom value or select an option from the menu. Placeholder text should always follow the pattern "Enter or select [thing(s)]", for example "Enter or select address". Ellipses are not needed. Use sentence casing.
+
+![ ](spec-images/combobox-2.png)
+
+#### Menu & Menu Items
+A list of options to select, displayed as an open state.*See [Select](/packages/nimble-components/src/select/specs/IxD.md) for more information.*
```

Comment:
It may be worth pointing out that the options used in the `Select` can have a value assigned to them that is distinct from the display value that has meaning to the `Select` (and the user). For the `Combobox` this is not the case, any value provided to an option is ignored.